ports/mimxrt10xx/common-hal/pwmio/PWMOut.c

```diff
@@ -138,33 +173,81 @@ pwmout_result_t common_hal_pwmio_pwmout_construct(pwmio_pwmout_obj_t *self,
 
     pwmConfig.prescale = self->prescaler;
 
+    DBGPrintf(&mp_plat_print, "\tCall PWM_Init\n");
     if (PWM_Init(self->pwm->pwm, self->pwm->submodule, &pwmConfig) == kStatus_Fail) {
         return PWMOUT_INVALID_PIN;
     }
 
+    // Disable all fault inputs
+    self->pwm->pwm->SM[self->pwm->submodule].DISMAP[0] = 0;
+    self->pwm->pwm->SM[self->pwm->submodule].DISMAP[1] = 0;
+
+    DBGPrintf(&mp_plat_print, "\tCall PWM_SetupPwm %p %x %u\n", self->pwm->pwm, self->pwm->submodule);
+    #if 0
```

Comment:
If this section isn't needed you can just remove it, unless there is value leaving it in the code.

Reply:
Thanks @gamblor21 

If you just remove this code, the pwm output does not happen,  as all of the channels are blocked by fault code. 

The alternative to the current code is probably what the Teensy code base does and at init time it clears out the fault conditions. 
```
void flexpwm_init(IMXRT_FLEXPWM_t *p)
{
	int i;

	p->FCTRL0 = FLEXPWM_FCTRL0_FLVL(15); // logic high = fault
	p->FSTS0 = 0x000F; // clear fault status
	p->FFILT0 = 0;
...
```
Note the register names are slightly different in the SDK file versus the PJRC file (imxrt.h)... But gives you the general idea of the which registers... 

Not sure why the SDK header files don't use the actual register names.  Like:
>>   55.8.50 Fault Control Register (FCTRL0)

Where the SDK has it as: FCTRL

Reply:
I was referring to the area in the `#if 0` which should never be included or am I missed something?

Reply:
> I was referring to the area in the `#if 0` which should never be included or am I missed something?

Sorry, I misunderstood.  

I removed the #if 0 code.   I did move the comments out into the other comment as to have it make more
sense.  I also left in a minor debug output (when debug enabled) which showed a couple more registers